aiohttp/helpers.py

```diff
@@ -225,6 +225,17 @@ def parse_remote_addr(forward):
     return remote[0], str(remote[1])
 
 
+def unquote_quote(segment):
```

Comment:
unquote_quoted sounds better. otherwise it's not simply to guess what it actually does (: